pkg/capture/capture.go

```diff
@@ -110,10 +110,10 @@ type Capture struct {
 
 	// Logged flows since creation of the capture (note that some
 	// flows are retained even after Rotate has been called)
-	flowLog *capturetypes.FlowLog
+	flowLog *FlowLog
```

Comment:
Moved back to the `capture` package because without the `GPPacket` type it now depends on `slimcap`. However, with the `Flows()` target for the API gone it's a non-issue.